src/client/actions/actions.js

```diff
@@ -0,0 +1,59 @@
+import C from './constants';
+
+function populateMovies(data) {
+    console.log(data);
+    if (data.errorcode === 404) {
+        return {
+            type: C.ERROR,
+            payload: data.message,
+        };
+    } else {
+        return {
+            type: C.POPULATE_MOVIES,
+            payload: data,
+        };
+    }
+}
+
+export function makeTitleSearch(value) {
```

Comment:
@vitkuz i think it will be cool to create something like 
```
makeSearch(type, value){
  ...
  return fetch(`https://netflixroulette.net/api/api.php?${type}=${value}`)
  ...
}
```

Reply:
@algbeta With new API urls are different so i think case suites perfectly